modell/src/test/kotlin/no/nav/tilbakekreving/Testdata.kt

```diff
@@ -154,3 +168,12 @@ fun behandling(
         foreslåVedtakSteg = ForeslåVedtakSteg(faktasteg, foreldelsesteg, vilkårsvurderingsteg, kravgrunnlagReferanse),
     )
 }
+
+private fun hentFødselsdatoFraFnr(fnr: String): LocalDate {
```

Comment:
Syntes vi heller skal hardkode datoen via LocalDate i stedet for å gjette det ut i fra fnr